### PR TITLE
{Compute} Update shared image gallery scenarios

### PR DESCRIPTION
**Pull request by olayemio**
**Description**

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
vm

**Comment by yungezz** [comment]
hi @qwordy , could you pls review the PR?

**Comment by qwordy** [comment]
Some checks were not successful

**Comment by qwordy** [comment]
Hi @olayemio , I haven't verified the examples that you updated can run successfully. Have you run them? Are you from shared image gallery service team? If both answers are true, I can assume the PR is good.

**Comment by olayemio** [comment]
Yes I'm from the product team and I have tried the examples

File: .vscode/launch.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
             "name": "Azure CLI Debug (Integrated Console)",
             "type": "python",
             "request": "launch",
-            "pythonPath": "${command:python.interpreterPath}",
+            "python": "${command:python.interpreterPath}",
```
Comment (qwordy) [inline .vscode/launch.json:8]: .vscode/launch.json is not relevant to this PR.

**Review by olayemio — COMMENTED** [review]
Deleted the .vscode/launch.json file in my commit

**Comment by qwordy** [comment]
> 
> 
> Deleted the .vscode/launch.json file in my commit

I suggest not change it. You can use "git reset" to reset it.

**Comment by olayemio** [comment]
@qwordy what's still pending

**Comment by qwordy** [comment]
> 
> 
> @qwordy what's still pending

I suggest not change .vscode/launch.json. You deleted it. Just keep it the original way. Do not change it.

**Comment by olayemio** [comment]
@qwordy I restored .vscode/launch.json file. How do I exclude it from this PR? I didn't try to make any changes to it so I have no idea why the PR pulled it in

**Review by qwordy — APPROVED** [review]

**Comment by qwordy** [comment]
No problem now. Thanks @olayemio 🙂

**Pull request merged**